src/tests/test_utils.cpp

```diff
@@ -635,22 +635,26 @@ class CPUID_Tests final : public Test {
 #if defined(BOTAN_TARGET_CPU_IS_X86_FAMILY)
 
          if(Botan::CPUID::has_sse2()) {
-            result.confirm("Output string includes sse2", cpuid_string.find("sse2") != std::string::npos);
+            if(!this->is_serialized()) {
```

Comment:
Is this necessary? Shouldn't we (if we registered as serialized) *always* be serialized?